nrf_802154_sl/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# Calculates relative path to the library to be uses taking into account
+# build type of the library, SoC type, floating point ABI, supported FEM type.
+# Note: Subdirectory layout is specific to nrf_802154_sl lib
+function(calculate_lib_path lib_path)
+  # Select build type of the library to link with
+  set(build_type_dir "Release")
+
+  # Select SoC
+  if(CONFIG_SOC_NRF52840)
+    set(soc_dir "nrf52840_xxAA")
+  elseif(CONFIG_SOC_NRF52833)
+    set(soc_dir "nrf52833_xxAA")
+  elseif(CONFIG_SOC_NRF52820)
+    set(soc_dir "nrf52820_xxAA")
+  elseif(CONFIG_SOC_NRF52811)
+    set(soc_dir "nrf52811_xxAA")
+  else()
+    message(WARNING "Selected SoC is not supported by the nrf_802154_sl lib.")
+  endif()
+
+  # Select floating ABI
+  if(CONFIG_FLOAT)
+    if(CONFIG_FP_HARDABI)
+        set(float_dir hard)
+    elseif(CONFIG_FP_SOFTABI)
+        set(float_dir softfp)
+    else()
+        assert(0 "Unreachable code")
+    endif()
+  else()
+	  set(float_dir soft)
+  endif()
+
+  # Select FEM type
+  set(fem_type_dir "three_pin_gpio")
```

Comment:
Is the intention that there will be a separate library for each FEM?

Reply:
Current state of SL is "yes", it has support for given FEM type at compile time. This can be either enabled in runtime (end then given type is used) or disabled. To have other FEM type other build is needed, we have such builds but don't release them at this time. This is not final solution, but we need to release now.

Reply:
Discussed F2F